javascript/builtins/SharedArrayBuffer.json

```diff
@@ -17,9 +17,8 @@
               }
             ],
             "chrome_android": {
-              "version_added": "60",
-              "version_removed": "63",
-              "notes": "Chrome disabled SharedArrayBuffer on January 5, 2018 to help reduce the efficacy of <a href='https://www.chromium.org/Home/chromium-security/ssca'>speculative side-channel attacks</a>. This is intended as a temporary measure until other mitigations are in place."
+              "version_added": "89",
```

Comment:
Assuming the 60-63 entry was correct, https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md doesn't have a rule supporting its removal. Can you instead turn this into an array (see Edge below) and add this new entry at the top?

Reply:
Sure, updated all 4x sections